data/json/items/armor.json

```diff
@@ -4834,6 +4834,24 @@
     },
     {
         "type" : "ARMOR",
+        "id" : "stillsuit",
+        "name" : "WS-47G moisture retention suit",
+        "weight" : 3481,
+        "color" : "dark_gray",
+        "covers" : ["TORSO", "ARMS", "LEGS"],
+        "symbol" : "[",
+        "description" : "The WebbStar moisture retention suit utilizes quantum cooling technology to prevent up to 30% of moisture loss through perspiration.  The suit is powered by the wearer's movement.",
```

Comment:
I think it's supposed to be powered in some other way, as the player can just stay still for a long time, like when sleeping while wearing this suit.